charts/IndicatorStore.ts

```diff
@@ -0,0 +1,59 @@
+import { fetchJSON } from "./Util"
+import { Indicator } from "./Indicator"
+import { BAKED_BASE_URL } from "settings"
+
+export class IndicatorStore {
```

Comment:
Have we thought about using an existing solution for this, like Redux or Relay or something?

Reply:
:+1: Will answer in main thread